programs/gauge/src/instructions/create_gauge.rs

```diff
@@ -79,8 +79,9 @@ pub fn handler(ctx: Context<CreateGauge>) -> Result<()> {
 
 impl<'info> Validate<'info> for CreateGauge<'info> {
     fn validate(&self) -> Result<()> {
-        assert_keys_eq!(self.quarry.amm_pool, self.amm_pool);
-        assert_keys_eq!(self.gauge_factory.rewarder, self.quarry.rewarder);
+        let quarry = self.quarry.load()?;
+        assert_keys_eq!(quarry.amm_pool, self.amm_pool);
```

Comment:
this is already in macro, so no need to assert here

Reply:
removed